circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/hive/RenameTableOperation.java

```diff
@@ -40,20 +40,27 @@ public RenameTableOperation(DropTableService dropTableService) {
    * renamed.
    */
   public void execute(CloseableMetaStoreClient client, Table from, Table to) throws Exception {
-    LOG
-        .info("Renaming table {}.{} to {}.{}", from.getDbName(), from.getTableName(), to.getDbName(),
-            to.getTableName());
+    LOG.info("Renaming table {}.{} to {}.{}", from.getDbName(), from.getTableName(), to.getDbName(), to.getTableName());
     Table fromTable = client.getTable(from.getDbName(), from.getTableName());
     Table toTable = client.getTable(to.getDbName(), to.getTableName());
+    String fromDatabaseName = fromTable.getDbName();
     String fromTableName = fromTable.getTableName();
     String toTableName = toTable.getTableName();
     String toDelete = toTableName + DELETE_ME;
     try {
+      fromTable.setDbName(toTable.getDbName());
       fromTable.setTableName(toTableName);
       toTable.setTableName(toDelete);
+      LOG
+          .info("Alter table {}.{} to {}.{}", toTable.getDbName(), toTableName, toTable.getDbName(),
+              toTable.getTableName());
       client.alter_table(toTable.getDbName(), toTableName, toTable);
-      client.alter_table(fromTable.getDbName(), fromTableName, fromTable);
+      LOG
+          .info("Alter table {}.{} to {}.{}", fromDatabaseName, fromTableName, fromTable.getDbName(),
```

Comment:
```suggestion
          .info("Renamed table {}.{} to {}.{}", fromDatabaseName, fromTableName, fromTable.getDbName(),
```

Reply:
that's wrong, I kept the "alter" but changed to "Altering"
If you run the tests the logs read like this and are very clear:
```
- Renaming table old_db.old_table to new_db.new_table
- Altering table new_db.new_table to new_db.new_table_delete_me
- Altering table old_db.old_table to new_db.new_table
- Dropping table new_db.new_table_delete_me
```